Dockerfile

```
@@ -0,0 +1,22 @@
+FROM ibm-semeru-runtimes:open-8-jdk-focal as builder
```

JDKとりあえずaarch64対応のあったibm-semeru-runtimesにしましたが、何にするかを考える必要あり。

---

from Misskey discussion
suggest1: https://hub.docker.com/layers/library/eclipse-temurin/8-jdk-focal/images/sha256-960759ff936cd4387618f837ad0643d44a10da5d17dd6730a116b4f55c7afd7a?context=explore

---

```suggestion
FROM eclipse-temurin:8-jdk-focal as builder
```
by https://misskey.niri.la/notes/9pl68srgaz